lib/takeaway.rb

```diff
@@ -0,0 +1,25 @@
+# deals with orders + cost
+require 'order'
+
+class Takeaway
+
+	def initialize(menu:, order: nil) #dependency injection - 'other class is injected into this class to enable menu class methods to be accessed'
```

Comment:
Metrics/LineLength: Line is too long. [144/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)